docs/examples/drag-and-drop/README.md

```diff
@@ -0,0 +1,258 @@
+# Drag and Drop
+## Drag and Drop to IPFS and Hypercore
+
+## Introduction
+This tutorial guides you through building a simple yet powerful web application that allows users to drag and drop files, publishing them directly to IPFS and Hypercore. Imagine Alice, an artist who wants to share her digital art easily. With this app, she can drag her artwork into the browser, and it's instantly available on the decentralized web.
+
+## Prerequisites
+- Basic knowledge of HTML, CSS, and JavaScript.
```

Comment:
Maybe we can link to MDN's intro to web programming for folks that don't already have these skills?

Reply:
Addded MDN links for HTML, CSS, and JavaScript